### PR TITLE
#165167110 Provide an enhancement to social authentication

### PR DESCRIPTION
**Pull request by devdbrandy**
#### What does this PR do?

~This provides an extra layer to register user from the payload received after making OAuth calls from the frontend.~

This provides an enhancement to social authentication by redirecting back to the client app with the user token.

#### Description of Task to be completed

- [x] Updated ENV config to include client redirect url
- [x] Removed additional middleware layer

#### How should this be manually tested

Follow guidelines on the [README](https://github.com/andela/apollo-ah-backend#set-up) to setup the project locally

*OR,* simply:

1. Clone repo by running the following command on your terminal:

```bash
git clone -b feature/165167110/oauth-payload-handle --single-branch \
https://github.com/andela/apollo-ah-backend.git
```

2. Navigate into the repo directory: `cd apollo-ah-backend`

2. Install dependency packages with: `npm install`

3. On the root directory, create a `.env.test` file from `.env.example` and provide required credentials

4. Run `npm test`

#### Any background context you want to provide

n/a

#### What are the relevant pivotal tracker stories

[#165167110](https://www.pivotaltracker.com/story/show/165167110)
[#164198183](https://www.pivotaltracker.com/story/show/164198183)

#### Screenshots

n/a

### CODE REVIEW

**Review by aanchirinah — APPROVED** [review]
LGTM

File: server/middlewares/socialCreate.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * Register a new user with oAuthentication
+ *
+ * @export
+ * @param {Request} request Request object
+ * @param {Response} response Response object
+ * @param {Function} next call to next middleware
+ * @returns {(Error|Function)} Proceeds to the next middleware else throws http error
+ */
+async function socialCreate(request, response, next) {
```
[the remark this answers is not in the document]
Comment (devdbrandy) [reply server/middlewares/socialCreate.js]: Resolved

**Review by davdwhyte87 — APPROVED** [review]
LGTM

**Review by devdbrandy — COMMENTED** [review]
Enhanced social authentication by Implementing a redirect back to the client app with the user token. By so doing, we necessarily do not need an additional library on the client app for social authentication.

**Pull request merged**